scripts/create-database.js

```diff
@@ -0,0 +1,55 @@
+// require the pg package
+const { Client } = require('pg');
+const path = require('path');
+
+// capture first command line argument passed to this script
```

Comment:
Would recommend removing all the comments from this, especially the one on line 42 as its obviously a suggestion and doesnt look good to anyone reading your repo for this first time